### PR TITLE
Feature/other user account wallet transfers

### PR DESCRIPTION
**Pull request by adilrime17**
I have completed this to the best of my knowledge of the app. Please review and test the implementation and let me know if I'm missing something.
Please test all the scenarios where Transfer component is used.
Best

### CODE REVIEW

File: src/common/components/wallet-ecency/index.tsx [hunk 1]
```diff
@@ -281,20 +282,20 @@ export const WalletEcency = (props: Props) => {
...
+    //     history: history,
+    //     label: '',
+    //     items: [{
+    //         label: _t('points.transfer'),
+    //         onClick: toggleTransfer
+    //     }, {
+    //         label: _t('points.promote'),
+    //         onClick: togglePromote
+    //     }, {
+    //         label: _t('points.boost'),
```
Comment (mghazanfar) [inline src/common/components/wallet-ecency/index.tsx:295]: @adilrime17 could we not add the condition here that you added in the below?

**Review by mghazanfar — CHANGES_REQUESTED** [review]

File: src/common/components/wallet-ecency/index.tsx (hunk 1 above)
Comment (adilrime17) [reply src/common/components/wallet-ecency/index.tsx:295]: To make it consistent with the other component which is wallet-hive, I have added it below so developers can relate the code just by seeing it.
Let me know if you still want to change this.

**Review by mghazanfar — APPROVED** [review]
LGTM

**Review by feruzm — APPROVED** [review]
lgtm

**Pull request merged**